spine-context-menu.html

```diff
@@ -0,0 +1,176 @@
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../vaadin-context-menu/vaadin-context-menu.html">
+
+<dom-module id="spine-context-menu">
+    <script>
+        {
+            /**
+             * An extension of the [`vaadin-context-menu`](https://github.com/vaadin/vaadin-context-menu)
+             * with enhanced positioning and keyboard control.
+             *
+             * Expands parent functionality adding an ability to adjust position
+             * relative to the viewport borders (i.e. when open near the viewport borders).
+             *
+             * This element is intended to handle the stacking contexts problem
+             * (for details see GitHub [issue](https://github.com/PolymerElements/paper-menu-button/issues/9#issuecomment-199478430),
+             *  and [PR](https://github.com/PolymerElements/iron-overlay-behavior/pull/155)).
+             *
+             * @customElement
+             * @polymer
+             * @demo demo/index.html
+             */
+            class SpineContextMenu extends Vaadin.ContextMenuElement {
+                static get is() {
+                    return 'spine-context-menu';
+                }
+
+                static get properties() {
+                    return {
+                        /**
+                         * If true, a dropdown content is rendered without overlapping the trigger element.
+                         */
+                        noOverlap: {
```

Comment:
Please add reference to `iron-overlay-behavior` where it's used.

Reply:
We don't use `iron-overlay-behavior` in this component at all.

Reply:
Please ask Vaadin team why they have `iron-overlay-behavior` as unused dependency.